package.json

```diff
@@ -36,13 +36,14 @@
     "101": "^0.18.0",
     "auto-debug": "^1.0.1",
     "boom": "^2.7.1",
+    "es6-error": "^2.1.0",
     "rollbar": "^0.5.3"
   },
   "devDependencies": {
     "code": "^1.4.0",
     "jsdoc": "^3.3.0-beta3",
-    "jshint": "^2.7.0",
     "lab": "^5.8.0",
```

Comment:
Since we re-wrote all the tests, it would have been nice to use the undoubtedly superior `mocha`.